tests/test_mc2020/test_mc2020_concrete_material_properties.py

```diff
@@ -0,0 +1,1363 @@
+"""Tests for fib MC2020 Chapter 14.6.1."""
```

Comment:
There are several lines that are not covered by tests. Can you please add tests to reach 100% coverage of the contribution?